packages/mobile/src/components/News.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {View, Text, Image, StyleSheet, ActivityIndicator} from 'react-native';
+
+const News = (props) => {
+	return(
+		<View style={ styles.newsItem }>
+			{ props.data.urlToImage ? <Image
```

Comment:
po co Ci ten pusty text? jesli chcesz zrobić warunek taki aby pokazywało Ci obrazek tylko jak jest urlToImage to możesz zrobić sobie nad return np const showImage = !!props.data.urlToImage; i tego {showImage && <Image .... albo { (props.data.urlToImage) && <Image ...